limited_area/limited_area.py

```diff
@@ -64,10 +64,40 @@ def __init__(self,
         elif self.boundary == 'search':
             # Possibly faster for smaller regions
             self.mark_neighbors = self._mark_neighbors_search
-        
-        
+
+        # Check to see that all given mesh files, simply exist
+        for mesh in files:
+            if not os.path.isfile(mesh):
+                print("ERROR: Mesh file was not found", mesh_file)
+                sys.exit(-1)
+
+        if len(files) == 1:
+            self.mesh = MeshHandler(files[0], 'r', *args, **kwargs)
+            if self.mesh.check_grid():
+                self.mesh.load_vars()
+            else:
+                print("ERROR:", self.mesh.fname, "did not contain needed mesh connectivity information")
+                sys.exit(-1)
+            self.meshes.append(self.mesh)
+
+        else:
+            self.mesh = MeshHandler(files.pop(0), 'r', *args, **kwargs)
+            if self.mesh.check_grid():
+                # Load the mesh connectivity variables needed to subset this mesh, and all the
+                # other ones
```

Comment:
This comment is misleading, as the other meshes are not loaded, but added to a list.